pytorch_pfn_extras/runtime/_runtime.py

```diff
@@ -11,8 +11,8 @@
 DeviceLike = Union[str, torch.device]
 
 if TYPE_CHECKING:
```

Comment:
Let's remove `if TYPE_CHECKING:` and replace `'Trainer'` with `Trainer` so that users can easily copy method signatures of base classes.

Reply:
Fixed. PTAL.

Reply:
FIxed.